web/sites/default/config/core.entity_view_display.taxonomy_term.community_actions.search_result.yml

```diff
@@ -29,3 +29,4 @@ content:
     third_party_settings: {  }
 hidden:
   field_fontawesome_icon: true
+  field_show_on_homepage: true
```

Comment:
Why are the new show on homepage fields showing up in your PR? Those are already in place in master. 🤔

Reply:
Revert to master

Reply:
No need to revert. While i don't know what caused it, it is not a breaking change. It just adds the field that should have been hidden to hidden. 🤔